### PR TITLE
Fix demo motortwit + fixate dependencies

### PR DESCRIPTION
**Pull request by yartem**
## What do these changes do?

Demo `motortwit` is currently outdated and fails in many places.

## Are there changes in behavior for the user?

No behaviour changes for the end users.

## Related issue number

No issue.

## Checklist

- [ ] I think the code is well written
- [ ] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [ ] Add a new news fragment into the `CHANGES` folder
  * name it `<issue_id>.<type>` (e.g. `588.bugfix`)
  * if you don't have an `issue_id` change it to the pr id after creating the PR
  * ensure type is one of the following:
    * `.feature`: Signifying a new feature.
    * `.bugfix`: Signifying a bug fix.
    * `.doc`: Signifying a documentation improvement.
    * `.removal`: Signifying a deprecation or removal of public API.
    * `.misc`: A ticket has been closed, but it is not of interest to users.
  * Make sure to use full sentences with correct case and punctuation, for example: `Fix issue with non-ascii contents in doctest text files.`

### CODE REVIEW

File: demos/motortwit/motortwit/db.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 user = t.Dict({
     t.Key('_id'): MongoId(),
     t.Key('username'): t.String(max_length=50),
-    t.Key('email'): t.Email(),
+    t.Key('email'): t.Email,
```
Comment (yartem) [inline demos/motortwit/motortwit/db.py:9]: `t.Email` is not a class but already a trafaret:
https://github.com/Deepwalker/trafaret/blob/40a521ae0f580a0e5a6f3168a59cbba350d5b3ba/tests/test_keys.py#L87

File: demos/motortwit/motortwit/generate_data.py [hunk 2]
```diff
@@ -21,7 +21,7 @@
 
 # TODO: validate using trafarets before insert
 async def insert_data(collection, values):
-    await collection.insert(values)
+    await collection.insert_many(values)
```
Comment (yartem) [inline demos/motortwit/motortwit/generate_data.py:24]: deprecated `insert`, `update` were removed in `motor` 2: https://motor.readthedocs.io/en/stable/migrate-to-motor-2.html

File: demos/motortwit/motortwit/main.py [hunk 3]
```diff
@@ -30,7 +30,8 @@ def setup_admin(app, mongo):
     resources = (MotorResource(m.user, db.user, url="user"),
                  MotorResource(m.message, db.message, url="message"),
                  MotorResource(m.follower, db.follower, url="follower"))
-    admin = aiohttp_admin.setup(app, admin_config_path, resources=resources)
+    admin = aiohttp_admin.setup(app, admin_config_path, 
+                                resources=resources, template_folder=TEMPLATES_ROOT)
```
Comment (yartem) [inline demos/motortwit/motortwit/main.py:34]: missing path to the templates directory

File: demos/motortwit/motortwit/templates/timeline.html [hunk 4]
```diff
@@ -17,11 +17,11 @@ <h2>{{ self.title() }}</h2>
           This is you!
         {% elif followed %}
           You are currently following this user.
-          <a class=unfollow href="{{ url('unfollow_user', parts={'username': profile_user.username})
+          <a class=unfollow href="{{ url('unfollow_user', username=profile_user.username)
```
Comment (yartem) [inline demos/motortwit/motortwit/templates/timeline.html:20]: it appears that the `url_for` does not accept the dict `parts` anymore, but instead it waits for `**kwargs`

File: demos/motortwit/motortwit/utils.py [hunk 5]
```diff
@@ -9,7 +9,7 @@
 
 def load_config(fname):
     with open(fname, 'rt') as f:
-        data = yaml.load(f)
+        data = yaml.load(f, Loader=yaml.FullLoader)
```
Comment (yartem) [inline demos/motortwit/motortwit/utils.py:12]: deprecated https://github.com/yaml/pyyaml/wiki/PyYAML-yaml.load(input)-Deprecation

File: demos/motortwit/setup.py [hunk 6]
```diff
@@ -18,16 +18,17 @@ def read_version():
...
+                    'pytz==2019.3',
+                    'bcrypt==3.1.7',
+                    'aiohttp_session==2.9.0',
+                    'aiohttp_admin>=0.0.3',
+                    # TODO: Update trafaret to 2.0.x (blocked by issue #487)
+                    'trafaret==1.2.0',
+                    'aiohttp_jinja2==1.2.0',
+                    'pyyaml==5.3.1',
+                    'motor==2.1.0',
+                    'faker==4.0.2']
```
Comment (yartem) [inline demos/motortwit/setup.py:31]: For sake of stability, let's fix the versions.

**Comment by Arfey** [comment]
thx

**Pull request merged**